examples/get_started/04_table_inputs.py

```diff
@@ -0,0 +1,145 @@
+"""
+4. PyGMT I/O: Table inputs
+==========================
+
+Generally, PyGMT accepts two different types of data inputs: tables and grids.
+
+- A table is a 2-D array with rows and columns. Each column represents a 
+  different variable (e.g., *x*, *y* and *z*) and each row represents a 
+  different record.
+- A grid is a 2-D array of data that is regularly spaced in the x and y
+  directions (or longitude and latitude). 
+
+In this tutorial, we'll focus on working with table inputs, and cover grid inputs
+in the following tutorials.
```

Comment:
Or do we plan to have multiple tutorials for grid inputs?
```suggestion
In this tutorial, we'll focus on working with table inputs, and cover grid
inputs in a separate tutorial.
```